From 46d9899f5a512d0e9b851cb55b06a27fc2b7a638 Mon Sep 17 00:00:00 2001
From: Takuya Wakazono <pastalian46@gmail.com>
Date: Wed, 1 Jan 2025 21:03:48 +0900
Subject: [PATCH] Explicitly declare function parameters for C23 compatibility

In C23, void foo() is equivalent to void foo(void). Therefore, functions
must explicitly declare their parameters.
---
 src/gbinder_writer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gbinder_writer.c b/src/gbinder_writer.c
index 1e93700..7308818 100644
--- a/src/gbinder_writer.c
+++ b/src/gbinder_writer.c
@@ -1324,7 +1324,7 @@ gbinder_writer_alloc(
     GBinderWriter* self,
     gsize size,
     gpointer (*alloc)(gsize),
-    void (*dealloc)())
+    void (*dealloc)(gpointer))
 {
     GBinderWriterData* data = gbinder_writer_data(self);
 
